patroni/dcs/__init__.py

```diff
@@ -585,50 +813,106 @@ class Cluster(NamedTuple):
 
     @staticmethod
     def empty() -> 'Cluster':
-        return Cluster(None, None, None, 0, [], None, SyncState.empty(), None, None, None)
+        """Produce an empty :class:`Cluster` instance."""
+        return Cluster(None, None, None, 0, [], None, SyncState.empty(), None, None, None, {})
```

Comment:
@CyberDem0n looks like there was a bug here? It was preventing the doctest from passing because some Citus tests write to cluster.workers. Should I create a separate PR for this?

Reply:
Bug is there are in fact 11 args to `Cluster` not 10...

Reply:
Hmmn seems to have broken tests, which passed locally. :thinking: 

https://github.com/zalando/patroni/actions/runs/5717044205/job/15489954467

Reply:
Yeah, with workers it is an interesting corner case.
It has a default value `{}`, which is mutated :(
The problem is introduced in #2652, when I changed `namedtuple` with overridden `__new__` method to `NamedTuple`.
I'll open a separate PR with the fix.

Reply:
Ah, thanks for taking that. I'll rebase when it's merged.